client/lib/abtest/active-tests.js

```diff
@@ -157,5 +157,15 @@ module.exports = {
 		},
 		defaultVariation: 'disabled',
 		allowExistingUsers: true
+	},
+
+	readerSearchOnFollowing: {
+		datestamp: '20170206',
+		variations: {
+			show: 50,
+			hide: 50
+		},
+		defaultVariation: 'hide',
+		allowExistingUsers: true
 	}
```

Comment:
Do we need to add `allowAnyLocale: true` to allow any language user to be admitted to the A/B test?